net/eds-bootstrap-enroll/Makefile

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright 2019 NXP
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=eds-bootstrap-enroll
+PKG_VERSION:=lsdk-1903
```

Comment:
There's a newer version.  It also doesn't match the actual tag: LSDK-19.03

Reply:
Thanks a lot!
The reason we chose LSDK-19.03 version was because this version was verified not only on NXP LSDK-19.03 release but also on NXP OpenWrt release. We could at least ensure it worked fine on Layerscape platforms. (I also did verification on i.MX6 platform personally.) We will also upgrade it to latest version when it's tested in future NXP OpenWrt release.